handy-win32src-0.95-patched/core/system.cpp

```diff
@@ -422,7 +422,7 @@ bool CSystem::IsZip(char *filename)
    UBYTE buf[2];
    FILE* fp;
    errno_t err;
-   if ((err = fopen_s(&fp, filename, "rb")) != 0) {
+   if ((err = fopen_s(&fp, filename, "rb")) == 0) {

```

Comment:
IMHO the variable **err** and the assignment in the **if ()** can be omitted as it is not used.

Reply:
Nice catch, I actually fixed it in my fork, I made this mistake a couple of time in this PR actually.
fopen return 0/NULL -> didnt work
fopen_s return 0 -> All fine
Pretty confusing ^^